test/mavsdk_tests/autopilot_tester_follow_me.cpp

```diff
@@ -363,68 +367,112 @@ void AutopilotTesterFollowMe::stream_velocity_only()
 	CHECK(distance_travelled < position_tolerance);
 }
 
-void AutopilotTesterFollowMe::rc_override_test(const float altitude_m)
+void AutopilotTesterFollowMe::rc_adjustment_test()
 {
-	const unsigned loop_update_rate = 50;
-	const float position_tolerance = 4.0f;
+	// CONFIGURATION
+	const unsigned loop_update_rate = 50; // [Hz]
+	const float follow_height_setting = 10.0f; // [m]
+	const float follow_angle_setting = 0.0f; // [deg]
+	const float follow_distance_setting = 10.0f; // [m]
+
+	// The constants below are copied from the "FlightTaskAutoFollowTarget.hpp" to get a reference point for
+	// how much change a RC adjustment is expected to bring for each follow me parameters
+
+	// [m/s] Speed with which the follow distance will be adjusted by when commanded with deflection via RC command
+	static constexpr float FOLLOW_DISTANCE_USER_ADJUST_SPEED = 2.0;
+	// [m/s] Speed with which the follow height will be adjusted by when commanded with deflection via RC command
+	static constexpr float FOLLOW_HEIGHT_USER_ADJUST_SPEED = 1.5;
+	// [rad/s] Angular rate with which the follow distance will be adjusted by when commanded with full deflection via RC command
+	static constexpr float FOLLOW_ANGLE_USER_ADJUST_SPEED = 1.5;
 
 	// Start with simulated target on the same plane as drone's home position
 	std::array<float, 3> start_location_ned = get_current_position_ned();
 	FollowTargetSimulator target_simulator(start_location_ned, getHome());
 
-	// Configure Min height of the drone to be "20 meters" above home and set the
-	// follow angle to 0 deg, which means following from the 'front' (0 deg)
+	// Set Follow Me parameters
 	FollowMe::Config config;
-	config.min_height_m = altitude_m;
-	config.follow_distance_m = 8.0f;
-	config.follow_angle_deg = 0.0f;
+	config.follow_height_m = follow_height_setting;
+	config.follow_distance_m = follow_distance_setting;
+	config.follow_angle_deg = follow_angle_setting;
 	CHECK(FollowMe::Result::Success == _follow_me->set_config(config));
 
-	// task loop
+	// Make sure the target is stationary and giving a clean data, to do a controlled test
```

Comment:
What do you mean by stationary here @junwoo091400 ? Steady state? Because the target is moving, i.e. not stationary :sweat_smile:

Reply:
No I am actually only using a single target sample to simulate a stationary target, only doing RC Adjustments! I will rename the variable so that it is more obvious 😉